File: myapp/parent/ErrorHandler.cs
using sfn_ut.sfn;
using sfn_ut.sfn.error;

namespace sfn_ut.myapp.parent;

public class ErrorHandler : SfnErrorHandler<ParentPayload> {
  public override SfnErrorHandlerInput<ParentPayload> Execute(SfnErrorHandlerInput<ParentPayload> payload) {
    return payload;
  }
}

File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/parent/ErrorHandler.java
package com.sfnut.myapp.parent;

import com.sfnut.sfn.SfnErrorHandler;
import com.sfnut.sfn.error.SfnErrorHandlerInput;

public class ErrorHandler extends SfnErrorHandler<ParentPayload> {
    @Override
    public SfnErrorHandlerInput<ParentPayload> execute(SfnErrorHandlerInput<ParentPayload> payload) {
        return payload;
    }
}
